drivers/timer/nrf_grtc_timer.c

```diff
@@ -300,7 +300,7 @@ uint64_t z_nrf_grtc_timer_get_ticks(k_timeout_t t)
 
 	grtc_ticks = t.ticks * CYC_PER_TICK;
 	abs_ticks = Z_TICK_ABS(t.ticks);
-	if (abs_ticks < 0) {
+	if (Z_IS_TIMEOUT_RELATIVE(t)) {
```

Comment:
I very much appreciate the self documenting macro here, the below comment is really unnecessary now here and elsewhere signaling its the relative timeout branch